overlay/files/system/oem/21_kcrypt.yaml

```diff
@@ -1,5 +1,10 @@
-name: "Update discovery plugins"
+name: "Kcrypt"
 stages:
+  rootfs:
+    - name: "Unlock encrypted volumes"
+      if: '[ ! -f "/run/cos/live_mode" ]'
+      commands:
+        - kcrypt unlock-all
   after-upgrade:
     - name: "Update plugins"
       if: "[ $(kairos-agent state get oem.found) == 'true' ]"
```

Comment:
not part of this PR, I know, but I think the code right after it can be dropped too

Reply:
we don't need to copy the plugins anymore into oem since .. a while (can't find the PR now)

Reply:
will drop it in a following PR, good to know!